frontend/src/metabase/lib/entities.js

```diff
@@ -576,6 +582,19 @@ export function createEntity(def: EntityDefinition): Entity {
       new EntityWrapper(object, dispatch);
   }
 
+  function trackAction(action, object, getState) {
+    try {
+      MetabaseAnalytics.trackEvent(
+        entity.name,
```

Comment:
I kind of liked having some sort of prefix to make it clear these events were automatic

Reply:
+100 please get these things into their own namespace. 

it's incredibly annoying to have to figure out where an event came from.